src/styles/block/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  cursor: pointer;
+  justify-content: center;
+  padding-top: 1px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
```

Comment:
Try to work on modificator for setting stars count, using `for` cycle

Reply:
Like stars--4

Reply:
really good idea, thanks